src/coord/src/coord.rs

```diff
@@ -5432,15 +5435,15 @@ impl Coordinator {
         // prevent us from incorrectly teaching those functions how to return errors
         // (which has happened twice and is the motivation for this test).
 
+        let compute_instance: ComputeInstanceId = 1;
```

Comment:
This seems like some random error will result in the future, where this number is not equal to some other number. Could/should we bundle this up somehow? Maybe a module-local constant?

Reply:
Well, this code will never run, it's only there to test the function signatures. I'll add a comment to clarify this.

Reply:
Oh I see. Not as clear from the GH diff snipping. Sorry!